### DISCUSSION TITLE
plugins options ?

### DISCUSSION THREAD

**Discussion by imclint21** [remarkjs/remark#1303] [Q&A]
Hi,

How to configure a specific plugins? how to pass arguments?

Regards

**Comment by ChristianMurphy** [comment]
Hey @imclint21 👋 

Do take some time to review the support guidelines. https://github.com/unifiedjs/.github/blob/main/support.md#support
Your question is super vague, its hard to give a good answer because there are hundreds of ways you could be using plugins.

For an overview of the ecosystem review: https://unifiedjs.com/learn/
If you have follow up questions expand on what you're looking for and what you've tried.
